site/docs/reference/working-logs-stats.md

```diff
@@ -83,3 +68,51 @@ acmeCo/anvils/logs:
         exclude:
           name: ['acmeCo/anvils/logs']
 ```
+
+## Available statistics
+
+Available statistics include information about the amount of data in inputs and outputs of each transaction. They also include temporal information about the transaction. Statistics vary by task type (capture, materialization, or derivation).
+
+A thorough knowledge of Flow's [advanced concepts](../concepts/README.md#advanced-concepts) is necessary to effectively leverage these statistics.
+
+`stats` collection documents include the following properties.
+
+### Shard information
+
+Each `stats` document begin with data about the shard processing the transaction.
+This information is important for tasks with multiple shards: it allows you to determine whether data throughput is
+evenly distributed amongst those shards.
+
+| Property | Description | Data Type | Applicable Task Type |
+|---|---|---|---|
+| `/shard` | Flow shard information| object | All |
+| `/shard/kind` | The type of catalog task. One of `"capture"`, `"derivation"`, or `"materialization"` | string | All |
+| `/shard/name` | The name of the catalog task (without the task type prefix) | string | All |
+| `/shard/keyBegin` | With `rClockBegin`, this comprises the shard ID. The inclusive beginning of the shard's assigned key range.  | string | All |
+| `/shard/rClockBegin` | With `keyBegin`, this comprises the shard ID. The inclusive beginning of the shard's assigned rClock range.  | string | All |
+
+### Transaction information
+
+`stats` documents include information about a transaction: its inputs and outputs,
+the amount of data processed, and the time taken.
+You can use this information to ensure that your Flow tasks are running efficiently,
+and that the amount of data processed matches your expectations.
+
+| Property | Description | Data Type | Applicable Task Type |
+|---|---|---|---|
+| `/ts` | Timestamp corresponding to the start of the transaction, rounded to the nearest minute | string | All |
+| `/openSecondsTotal` | Total time that the transaction was open before starting to commit | number | All |
+| `/txnCount` | Total number of transactions represented by this stats document. Used for reduction; will always be `1`. | integer | All |
```

Comment:
nit: `will always be 1` might be confusing for people, since by the time they see a stats document in either a derivation or materialization, it's very likely to be greater than 1 due to reductions.